lib/LDAPGroupManager.php

```diff
@@ -82,15 +92,27 @@ public function respondToActions() {
 	 * @return string|null
 	 */
 	public function createGroup($gid) {
+		$adminUser = $this->userSession->getUser();
+		$requireActorFromLDAP = $this->configuration->isLdapActorRequired();
+		if ($requireActorFromLDAP && !$adminUser instanceof IUser) {
```

Comment:
```suggestion
		if ($requireActorFromLDAP && !($adminUser instanceof IUser)) {
```
I was unsure of the precedence between operators here so I expect others might